sway/ipc-json.c

```diff
@@ -1002,6 +1003,17 @@ json_object *ipc_json_describe_input(struct sway_input_device *device) {
 		}
 	}
 
+	if (device->wlr_device->type == WLR_INPUT_DEVICE_POINTER) {
+		struct input_config *ic = input_device_get_config(device);
+		float scroll_factor = 1.0f;
+		if (ic != NULL && !isnan(ic->scroll_factor) && 
+				ic->scroll_factor != FLT_MIN) {
+			scroll_factor = ic->scroll_factor;
+		}
+		json_object_object_add(object, "scroll_factor", 
+				json_object_new_int(scroll_factor));
```

Comment:
Ah, one last thing: this should be `json_object_new_float`, not `new_int`

Reply:
Oh, my bad 😅